polling_stations/apps/pollingstations/models.py

```diff
@@ -141,20 +141,26 @@ def get_absolute_url(self):
 class CustomFinderManager(models.Manager):
     def get_custom_finder(self, geocoder, postcode):
         try:
-            finder = self.get(pk__in=[geocoder.get_code("lad")])
-            finder.message = _(finder.message)
-            """
-            EONI's poling station finder requires postcode to have a space :(
-            http://www.eoni.org.uk/Offices/Postcode-Search-Results?postcode=BT5+7TQ
-            will produce a result, whereas
-            http://www.eoni.org.uk/Offices/Postcode-Search-Results?postcode=BT57TQ
-            will not.
-
-            We might need to take a more sophisticated approach as we add more custom finders
-            that accept postcodes (e.g: a postcode format flag in the database).
-            At the moment I only have this one to work with.
-            """
-            finder.encoded_postcode = urllib.parse.quote(Postcode(postcode).with_space)
+            code = geocoder.get_code("lad")
+            if code.startswith("N"):
+                finder = self.get(pk="N07000001")
```

Comment:
Thought I'd present pros and cons of a couple of alternative approaches here to stop slack swallowing them. The current option I just did as 'the simplest way to make it work™', an am not committed to. I think option 2 might be best, I think I'd want to write some tests so that if the councils were updated and no new CustomFinder object was created we'd know about it. 

#### Option 1
The approach in this commit is obviously hacky, but is easy to change from and leaves all the existing code to handle custom finders in place. I've checked it with `BT15 3JX` on the front end and the API, and as long as Local Authorities codes in NI keep being prefixed with an `N` is should keep working.

#### Option 2 Make a CustomFinder object for the NI Local Auths
Do [this](https://github.com/DemocracyClub/UK-Polling-Stations/blob/697437a02b87959116e45a3e41accd34a56184a4/polling_stations/apps/pollingstations/migrations/0013_customfinders.py) but `for gss in ["N09000001", "N09000002"....]` for the 11(?) Local Authorities. Pros: we can still introduce new CustomFinders easily. The only con I can think of is the need to keep on top of any changes to the codes and make the appropriate migrations. Given this is a rare occurrence it won't be much work, but it would be easy to forget to do. 

#### Option 3 Bin CustomFinder Model 
We could get rid of the CustomFinder completely and just insert code to check for a NI local authority and hard code the `context['custom']` [here](https://github.com/DemocracyClub/UK-Polling-Stations/blob/dcbdfa247d8a98ed217528a62de0ac77a1661bcc/polling_stations/apps/data_finder/views.py#L175). Which I think would work.
The pros of this are that we'd be reducing code since the only custom finder we handle is EONI. The cons are we might want to support others in the future, and I'm not certain it's quite as straightforward as I've described.

Reply:
I reckon the way you've done this is fine. You can probably bin the CustomFinder model, but its not _harming_ anything to keep it there.

Reply:
I'm nervous of getting into the weeds with that, and it should probably be a separate PR anyway, so will leave it as is.